Makefile

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2025 Norbert Melzer <timmelzer@gmail.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+CUE_DIR := internal/ci
+WORKFLOWS_DIR := .github/workflows
+
+SPDX_HEADER := \# SPDX-FileCopyrightText: 2025 Norbert Melzer <timmelzer@gmail.com>\n\#\n\# SPDX-License-Identifier: CC0-1.0\n\n\# This file is generated from internal/ci/workflows.cue\n\# Do not edit directly. Run make generate-workflows to regenerate.\n\n
+
+.PHONY: generate-workflows
+generate-workflows:
```

Comment:
Can you please add dependencies for the actually generated files, and a target for each file that shall be generated?